OpenEdXMobile/res/values/styles.xml

```diff
@@ -323,6 +323,7 @@
         <item name="editTextColor">@color/edx_brand_primary_base</item>
         <item name="android:editTextColor">@color/edx_brand_primary_base</item>
         <item name="android:textColorHint">@color/edx_brand_primary_base</item>
+        <item name="android:fontFamily">?attr/fontSemiBold</item>
```

Comment:
why we are adding this here?

Reply:
As per Figma design, android `searchview` hint is semibold.